src/GiGraph.Dot/Extensions/DotGraphExtension.cs

```diff
@@ -32,21 +33,17 @@ public static class DotGraphExtension
         ///     The syntax rules to use.
         /// </param>
         /// <param name="encoding">
-        ///     The encoding to use for the output text.
+        ///     Unused. Kept for backward compatibility.
         /// </param>
         public static string Build(this DotGraph graph, DotFormattingOptions formattingOptions = null,
             DotSyntaxOptions syntaxOptions = null, DotSyntaxRules syntaxRules = null, Encoding encoding = null)
```

Comment:
The encoding may now be removed since it is not used any more here.

Reply:
Well, removing a parameter from a public method is both a source and binary breaking change. As a consumer of libraries, I like when they follow [Semantic Versioning](https://semver.org) and don't introduce breaking changes in minor version updates. That's why I intentionally left the parameter untouched.

Reply:
Makes sense. However, I'm not actively developing the package now. I think I got to a point where I implemented all that I planned to implement, and don't currently have any more new ideas to include, unless there appear requests or suggestions like yours ;). So far I haven't used that versioning approach (but maybe it was wrong of me). I changed versions based on the amount of changes or new functionalities introduced, without paying much attention to backward compatibility. Thanks for bringing it up though, I'll consider this approach in the future!